exporter/tanzuobservabilityexporter/README.md

```diff
@@ -46,21 +54,51 @@ exporters:
 service:
   pipelines:
     traces:
-      receivers: [examplereceiver]
-      processors: [batch]
-      exporters: [tanzuobservability]
+      receivers: [ examplereceiver ]
+      processors: [ batch ]
+      exporters: [ tanzuobservability ]
 ```
 
 ### Advanced Configuration
 
-This exporter uses [queuing and retry helpers](https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/exporterhelper/README.md) provided by the core OpenTelemetry Collector. The `retry_on_failure` and `sending_queue` features are enabled by default, but can be disabled using the options below.
-
-* `retry_on_failure` [Details and defaults here](https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/exporterhelper/README.md#configuration). Enabled by default.
+#### Processor
+
+The memory limiter processor is used to prevent out of memory situations on the collector. It allows performing periodic
+checks of memory usage if it exceeds defined limits will begin dropping data and forcing GC to reduce memory
+consumption.
+
+* `memory_limiter` [Details and defaults here](https://github.com/open-telemetry/opentelemetry-collector/blob/main/processor/memorylimiterprocessor/README.md)
+    * `check_interval` 0s by default.
```

Comment:
I'm a little torn on whether it would be better to explain what the `0` defaults mean here or just omit them and let folks go the memorylimiterprocessor README instead - any thoughts?

Reply:
I think it's a good idea to let them know what default values mean. I'll update.